k-means-minus-two/test/test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+source ../../test-utils.sh
+
+rm -f -R cmd_del
+rm -f -R cmd
+rm -f -R .build
+
+log "-------------------------------------------------------"
+log "Test for k-means minus two"
+run_bigmler whizzml --package-dir ../ --output-dir ./.build
+# creating the resources needed to run the test
+run_bigmler --train s3://bigml-public/csv/diabetes.csv --no-model\
+            --project "Whizzml examples tests" --output-dir cmd/pre_test
+
+# building the inputs for the test
+prefix='[["dataset", "'
+suffix='"], ["k", 5], ["l", 10], ["threshold", 0.95], ["maximum", 5]]'
+text=''
+cat cmd/pre_test/dataset | while read dataset
+do
+echo "$prefix$dataset$suffix" > "test_inputs.json"
+done
+
+log "Testing k-means minus two script ----------------------------------"
+# running the execution with the given inputs
+run_bigmler execute --scripts .build/scripts --inputs test_inputs.json \
+                    --output-dir cmd/results
+# check the outputs
+declare file="cmd/results/whizzml_results.json"
+
+# check the outputs
+declare file="cmd/results/whizzml_results.json"
+declare regex="\"outputs\": \[\[\"output-list\",\
```

Comment:
i almost sure you need to double the backslashes there for this regexp to work everywhere (old versions of bash seem to have supported a more lenient syntax)... so that'd be "\\[" instead of "\["...

Reply:
oh, no, sorry... the failure in the test is because we're not outputting outputs_list anymore

Reply:
So you are saying if you change the outputs, you need to change the test. Oops. Fixed!